DependencyInjection/MonologExtension.php

```diff
@@ -699,6 +699,25 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
             ));
             break;
 
+        case 'redis':
+            if(class_exists('\Redis')){
+                $client = new Definition('\Redis');
```

Comment:
could you combine my pullrequest with yours? Then we have a monologHandler for Redis and Predis that works with services as with just redis-host, port, ... etc...
https://github.com/symfony/monolog-bundle/pull/225/files

Then I can close my pullrequest.